transport/src/commonMain/kotlin/com/genesys/cloud/messenger/transport/core/Message.kt

```diff
@@ -13,7 +13,7 @@ data class Message(
     val state: State = State.Idle,
     val type: String = "Text",
     val text: String? = null,
-    val timeStamp: String? = null,
+    val timeStamp: Long? = null,
```

Comment:
most of the `Message` properties seem self-explanatory, but with the change of `timeStamp` from `String` (which would be an ISO timestamp) to `Long` maybe it's a good idea to document it. Maybe something like:

`*  @param timeStamp the time when the message occurred represented in Unix epoch time, the number of milliseconds since January 1, 1970 UTC.`

Reply:
Could I suggest slight wording changes to the KDoc comments? I'm also not sure if we need to call out in the param description that something is optional as it seems like that's already covered by the object type. But, I could be swayed if there is prior art for doing so.

```
  *  @property id a unique message identifier.
  *  @property direction the direction in which the message was sent.
  *  @property state the current message state.
  *  @property type the message type. The default type is "Text".
  *  @property text the text payload of the message.
  *  @property timeStamp the time when the message occurred represented in Unix epoch time, the number of milliseconds since January 1, 1970 UTC.
  *  @property attachments a map of [Attachment] files to the message. Empty by default.
```